specs/ChainSpec/Consensus.md

```diff
@@ -0,0 +1,146 @@
+# Consensus
+
+The fields in the Block header relevant to the consensus process are:
+
+```rust
+struct BlockHeader {
+    ...
+    prev_hash: BlockHash,
+    height: BlockHeight,
+    epoch_id: EpochId,
+    last_final_block_hash: BlockHash,
+    approvals: Vec<Approval>
```

Comment:
We don't need full approvals here, only `(account_id, signature)` tuples. Remaining information is already present in the block header.

Reply:
Replacing with `Vec<Option<Signature>>`